src/tests/suite/daos_container.c

```diff
@@ -2048,6 +2048,7 @@ expect_co_get_attr_access(test_arg_t *arg, uint64_t perms, int exp_result)
 	const char	*name = "AttrName";
 	size_t		 val_size = TEST_MAX_ATTR_LEN;
 	char		 value[val_size];
+	void		*valptr = &value;
```

Comment:
This change is unrelated to the patch, but the patch is exposing a bug in our test suite. One cannot pass &value to cont_get_attr() below since it is a array allocated on the stack. We need another indirection.